src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx

```diff
@@ -61,19 +64,34 @@ const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
             <Typography>{weeksPipe(proposedSolution.timelineImpact)}</Typography>
           </Grid>
           <Grid item>{proposedSolution.approved && <Chip label="Approved" color="success" />}</Grid>
-          {showDeleteButton && onDelete !== undefined && (
-            <Grid item>
-              <Button
-                color="error"
-                variant="outlined"
-                onClick={() => {
-                  onDelete(proposedSolution);
-                }}
-              >
-                <DeleteIcon />
-              </Button>
+          <Grid item>
+            <Grid container direction="row" spacing={2}>
+              <Grid item>
+                <Button
+                  color="info"
```

Comment:
this should be green, not blue:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/fbc0368b-e2a0-4434-b12f-6a492c3a0d7c)